WPF/Model/Task.cs

```diff
@@ -184,17 +184,37 @@ public override bool RemoveWorker(User worker)
         #region Dependencies
         public void UpdateDependencies()
         {
-
+            
         }
-
+        //Creates a dependency, the parameter is the Root, and the current task is the dependent. 
         public void AddDependency(Task dependency)
         {
-
+            string query = "EXEC CreateDependency @RootId, @DependentId";
+            SqlCommand command = OpenConnection(query);
+            command.Parameters.AddWithValue("@RootId", dependency.Id);
+            command.Parameters.AddWithValue("@DependentId", this.Id);
+            command.ExecuteNonQuery();
+            CloseConnection();
```

Comment:
And remember to call NotifyUpdate() to update subscribers that a change happened